sklearn/preprocessing/data.py

```diff
@@ -1730,13 +1728,14 @@ def _transform_selected(X, transform, selected="all", copy=True):
 
 
 class OneHotEncoder(BaseEstimator, TransformerMixin):
-    """Encode categorical integer features using a one-hot aka one-of-K scheme.
+    """Encode ordinal integer features using a one-hot aka one-of-K scheme.
```

Comment:
I used "ordinal" to indicate that the actual set of values do mean something (eg the values [1,4,5] means that there are also categories 2 and 3 which are not present in the data), but I agree that this is not the same as being ordered. 
That said, I also find 'categorical' misleading given the above behaviour.